tests/manual-test-cases/Group19-DRS-Disabled/19-3-DRS-Disabled-Placement.md

```diff
@@ -11,35 +11,29 @@ The current placement strategy is to avoid bad host selection, instead of select
 
 # Environment:
 This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
-* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
-* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+* 1 vCenter host with 1 cluster containing 4 ESX hosts with DRS disabled.
 
 In addition, this test should be run in multi-ESX-host and single-ESX-host cluster topologies.
 
 See https://confluence.eng.vmware.com/display/CNA/VIC+ROBO for more details.
 
 # Test Steps:
-1. Deploy a ROBO Advanced vCenter testbed for both environments above
-2. Run a few dummy VMs on the host(s) in the vCenter cluster meant for VCH installation. This ensures that the hosts have varying resource utilization.
-3. Install a VCH on a particular cluster on vCenter - see note in [Environment](#environment)
-4. Deploy containers that will consume resources predictably (e.g. the `progrium/stress` image)
-5. Measure cluster metrics and gather resource consumption
-6. Create and run regular containers such as `busybox`
-7. Create and run enough containers to consume all available cluster resources
-8. Attempt to create and run more containers
-9. Delete some containers
-10. Create and run a few containers
-11. Delete the VCH
+1. Deploy a vCenter testbed with DRS disabled
+2. Install the VIC VCH appliance with compute resource set to the cluster
+3. Deploy 2 containers that will consume resources predictably (e.g. the `progrium/stress` image)
+4. Measure cluster metrics and gather resource consumption
+5. Create a normal container using the `busybox` image
+6. Relocate the `busybox` container to the same host as the VCH
```

Comment:
we are not intending to manually relocate this right?  this is worded like it is manual, but I believe you mean for the new placement algorithms to do it automatically right?

Reply:
Yes, we are intending to manually relocate. Since vSphere's create would likely place it on the least-loaded host (even without DRS enabled), we force it onto a less-ideal host (the one already containing the VCH vs. the empty/unused host) so that we can assert that it was automatically relocated to the ideal host when powered on.